gatsby-config.js

```diff
@@ -80,6 +80,23 @@ module.exports = {
       : []),
     'gatsby-plugin-typescript',
     'gatsby-plugin-emotion',
+    ...(process.env.GTM_TRACKING_ID
+      ? [
+          {
+            resolve: 'gatsby-plugin-google-tagmanager',
+            options: {
+              id: process.env.GTM_TRACKING_ID,
+              includeInDevelopment: false,
+              routeChangeEventName: 'e_routeChange',
+              defaultDataLayer: {
+                environmentName: process.env.FLAMELINK_ENV,
+                pageType: 'help',
```

Comment:
I see Lloyd only added help and inapp, we can make this pageType: page I think

Reply:
good idea yes. was wondering about that. will update